uaclient/tests/test_cli_auto_attach.py

```diff
@@ -38,7 +38,9 @@ def fake_instance_factory(self):
         m_instance.identity_doc = "pkcs7-validated-by-backend"
         return m_instance
 
-    @pytest.mark.parametrize("cloud_type", ("awslookalike", "azure2", "!aws"))
```

Comment:
I'd actually prefer to keep this one in, as it ensures we aren't testing for a _prefix_ of "aws".